packages/nextjs/next.config.js

```diff
@@ -11,6 +11,7 @@ const nextConfig = {
   },
   webpack: config => {
     config.resolve.fallback = { fs: false, net: false, tls: false };
+    config.externals.push("pino-pretty", "lokijs", "encoding");
```

Comment:
why is this needed?

Reply:
https://github.com/WalletConnect/walletconnect-monorepo/issues/1908